Marlin/pins_RIGIDBOARD.h

```diff
@@ -119,7 +119,7 @@
   #undef  SD_DETECT_PIN
   #define SD_DETECT_PIN 22
 
-#elif ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER)
+#elif ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) || ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
```

Comment:
When `REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER` is enabled, `REPRAP_DISCOUNT_SMART_CONTROLLER` is automatically also enabled.